ros_ws/src/car_control/src/drive_parameters_multiplexer.cpp

```diff
@@ -41,6 +51,11 @@ void DriveParametersMultiplexer::onUpdate(DriveParametersSource* source,
     this->m_last_updated_source = source;
 }
 
+void DriveParametersMultiplexer::driveModeCallback(const std_msgs::Int32::ConstPtr& drive_mode_message)
+{
+    this->m_drive_mode = (DriveMode)drive_mode_message->data;
```

Comment:
Here you could add an assertion to make sure the given data is indeed a valid DriveMode value.